nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -780,11 +780,8 @@ where
     match result {
         Ok(result) => {
             let output = result.into_inner();
-            tokio::fs::write(
-                path.join(format!("{command}.txt")),
-                format!("{output:?}"),
-            )
-            .await?;
+            let json = serde_json::to_string(&output)?;
+            tokio::fs::write(path.join(format!("{command}.txt")), json).await?;
```

Comment:
Context on failure would be nice as well

```suggestion
            tokio::fs::write(path.join(format!("{command}.txt")), json).await.with_context(|| format!("failed to write output of {command} to file"))?;
```

Reply:
Fixed in [e610f80](https://github.com/oxidecomputer/omicron/pull/7462/commits/e610f804331a124f840cc2ad477c05b4e52f9537)